homework/task_2.py

```diff
@@ -5,6 +5,4 @@
       x = n % 10
       summa = summa + x
       n = n // 10
```

Comment:
Можно записать покороче.
```suggestion
      summa += n % 10
      n //= 10
```